getting_started.md

```diff
@@ -80,46 +96,45 @@ b = rand(2)
 
 ### Step 4: Define a loss function
 
-A loss function evaluates a machine learning model's performance. In other words, it measures how far the model is from its target prediction. Flux enables you to define your own custom loss function or you can use one of the [Loss Functions](https://fluxml.ai/Flux.jl/stable/training/training/#Loss-Functions-1) that Flux provides. 
+A loss function evaluates a machine learning model's performance. In other words, it measures how far the model is from its target prediction. Flux lets you define your own custom loss function, or you can use one of the [Loss Functions](https://fluxml.ai/Flux.jl/stable/training/training/#Loss-Functions-1) that Flux provides. 
 
-For this example, define a custom loss function:
+For this example, we'll define a loss function that measures the squared distance from the predicted output to the actual output:
 
 ```julia
 function loss(x, y)
   ŷ = model(x)
-  sum((y .- ŷ).^2)
+  sum((y - ŷ).^2)
```

Comment:
```suggestion
  sum((y .- ŷ).^2)
```
I understand the want to avoid strange behaviour with broadcasting differently sized operands, but we should be consistent and removing the dot will break broadcast fusion between `-` and `^` (inconsequential here, but a big deal for more expensive forward passes).

Reply:
Actually, I just totally forgot about those. Thanks for the catch.